front/pages/poke/templates/[tId].tsx

```diff
@@ -554,7 +554,7 @@ function TemplatesPage({
       description: "",
       handle: "",
       presetInstructions: "",
-      presetModelId: GPT_4_TURBO_MODEL_CONFIG.modelId,
+      presetModelId: CLAUDE_4_SONNET_DEFAULT_MODEL_CONFIG.modelId,
       presetTemperature: "balanced",
```

Comment:
If we remove the temperature field from UI we should also cleanup the form no? 🙏🏻